packages/cascara/src/components/Dashboard/DashboardDX.fixture.js

```diff
@@ -66,7 +66,27 @@ const WIDGETS = [
     data: undefined,
     // keys: ['country', 'fries', 'curry'],
     // rowAction: (obj) => console.log(obj), // Without a rowAction defined, no row action will show. Note that the function gets the original object passed to it.
-    title: 'List',
+    title: 'List (simple)',
+    widget: 'list',
+  },
+  {
+    data: undefined,
+    keys: ['not helpful', 'no data', 'helpful', 'month'],
```

Comment:
it looks like Cascara now knows about the context of the app (no helpful, no data) etc. Is this correct? Cascara shouldn't contain logic about the app usage

Reply:
No, no, @odiseo42 , Cascara doesn't know about the logic of the app or the context 
 This is a fixture, this is an example object of how the final user (US) can use the widgets.
 
 But it's mostly to help us to develop in Cascara using cosmos. Fixtures feed the Cosmos app. 
 
 
<img width="639" alt="image" src="https://user-images.githubusercontent.com/80282588/201171995-0567e405-344e-4871-a4c9-7c743e402b2e.png">
<img width="1496" alt="image" src="https://user-images.githubusercontent.com/80282588/201172034-80b9f8bb-8464-427a-a47a-bbad763d9fe4.png">